internal/config/testdata/nginx-agent.conf

```diff
@@ -11,7 +11,8 @@ watchers:
 data_plane_config:
     nginx:
         reload_monitoring_period: 30s
-        treat_warnings_as_error: false
+        treat_warnings_as_errors: false
+        exclude_logs: "/var/log/nginx/error.log,/var/log/nginx/access.log"
```

Comment:
can we make this consistent with config_dirs ?

Reply:
we can, I was just keeping it consistent with what we had in v2 but can change it

Reply:
Changed